mne/epochs.py

```diff
@@ -775,6 +775,46 @@ def drop_log_stats(self, ignore=['IGNORED']):
         """
         return _drop_log_stats(self.drop_log, ignore)
 
+    def apply_reject(self, reject=None, flat=None):
```

Comment:
thinking about this why not adding the reject and flat to drop_bad_epochs method?